src/Commands/Plumbing/Merge/MergeTree.hs

```diff
@@ -9,9 +9,9 @@ import qualified Data.ByteString.Char8      as B
 cmdMergeTree :: Command
 cmdMergeTree (o:a:b:_) = do
   let (hashO, hashA, hashB) = (B.pack o, B.pack a, B.pack b)
-  let treeO = loadTree hashO
-  let treeA = loadTree hashA
-  let treeB = loadTree hashB
+  let treeO = loadObject hashO
+  let treeA = loadObject hashA
+  let treeB = loadObject hashB
```

Comment:
I feel these lines repeat themselves unnecessarily. Might be worth merging with the one above, too. What do you think?

```suggestion
  let (treeO:treeA:treeB:_) = map loadObject [hashO, hashA, hashB]
```

Reply:
That leads to this warning:

<img width="589" alt="Captura de Tela 2023-03-08 às 16 05 22" src="https://user-images.githubusercontent.com/12701580/223812669-7ea23ce5-7b26-434d-882a-f54b00ec01c2.png">

I think ghc isn't smart enough to know that this list will always have only three elements...

Reply:
Fair enough. We can at least lower the number of lines in this `do` block as such:

```haskell
cmdMergeTree :: Command
cmdMergeTree (o:a:b:_) = do
  let (treeO, treeA, treeB) = (hashToTree o, hashToTree a, hashToTree b)
      hashToTree = loadTree . B.pack

  merged <- mergeTree treeO treeA treeB
  putStrLn $ B.unpack . hashObject $ merged
```

What do you think? I'll leave it as a suggestion, but approve the PR. :P

Reply:
This works. Done in 93b5c6008210623970dbef81c0a33a2a690e450a.

Probably this code could be a lot cleaner using things like that.